lib/improver/tests/utilities/cube_manipulation/test_ConcatenateCubes.py

```diff
@@ -217,7 +216,7 @@ def test_missing_master_coord(self):
         self.cube.remove_coord("time")
         msg = "Master coordinate time is not present"
         with self.assertRaisesRegex(ValueError, msg):
-            self.plugin.process(self.cube)
+            self.plugin.process([self.cube, self.cube])
```

Comment:
For subsequent reviewers, this change is necessary as the plugin now returns a single cube prior to reaching the check that throws this error, hence the need to now test it with a cubelist.